data/human/deep missions.txt

```diff
@@ -4088,7 +4088,7 @@ mission "Hjlod Remembers Nifel"
 	on offer
 		dialog
 			`Your inbox hums with a new message from Nifel.`
-			`	"Dear <first>,`
+			`	"Dear <first>,"`
 			`	I never properly thanked you for saving my life all those years ago. If you're ever in the neighborhood, please stop by <planet>.`
 			`				Signed, Hjlod"`
```

Comment:
Ditto.

Reply:
Shouldn't `"Dear <first>,"` loose it's indentation?